docs/dev/feed-cvo-custom-graphs.md

```diff
@@ -0,0 +1,61 @@
+You might come across some scenarios when you want to give cluster verison operator some custom update graph which is not possible through the public instance of OpenShift update service. 
+
+Here are the steps
+
+### Step-1
+
+Create a JSON file with the custom graph.
+
+Example:
+```json
+{
+  "edges": [
+    [0, 1]
+  ],
+  "nodes": [
+    {
+      "payload": "registry.build01.ci.openshift.org/ci-ln-2573f6b/release@sha256:e8cbb6d5e36028ab2336308929f89aca5e49951794c6aa559fca1f24fda29d11",
+      "version": "4.8.12",
```

Comment:
nit: no trailing commas in JSON; you only want commas between entries in the object or array.